src/Core/tests/DeviceTests.Shared/HandlerTests/HandlerTestBasementOfT.cs

```diff
@@ -118,6 +158,27 @@ protected Task SetValueAsync<TValue>(IView view, TValue value, Action<THandler,
 			Assert.Equal(expectedPlatformValue, values.PlatformViewValue);
 		}
 
+		async protected Task ValidatePropertyInitValue<TValue, TCustomHandler>(
+			IView view,
+			Func<TValue> GetValue,
+			Func<TCustomHandler, TValue> GetPlatformValue,
+			TValue expectedValue,
+			TValue expectedPlatformValue)
+			where TCustomHandler : IElementHandler, new()
+		{
+			var values = await GetValueAsync(view, (TCustomHandler handler) =>
+			{
+				return new
+				{
+					ViewValue = GetValue(),
+					PlatformViewValue = GetPlatformValue(handler)
+				};
+			});
+
+			Assert.Equal(expectedValue, values.ViewValue);
+			Assert.Equal(expectedPlatformValue, values.PlatformViewValue);
```

Comment:
I'm seeing the only difference between this test and the previous one is that this one has sa separate `expectedPlatformValue` param to compare against. Why is that? Why do we need both?

Reply:
I copied both existing assert methods do be complete, but there only appears to be a single instance where this extra parameter is used:

https://github.com/dotnet/maui/blob/95583f7d8527103bd34f58e3000d42b738c35e9e/src/Core/tests/DeviceTests/Handlers/DatePicker/DatePickerHandlerTests.Windows.cs#L12-L25

Looks like the date picker is one where we set a common format, but then the native platform has a different format.